src/api/app/controllers/application_controller.rb

```diff
@@ -460,14 +460,14 @@ def shutup_rails
   end
 
   def set_nobody
-    User.current = User.find_nobody!
+    User.set_current(User.find_nobody!)
   end
 
   def set_influxdb_data
-    anonymous = User.current_login == User::NOBODY_LOGIN
+    anonymous = User.possibly_nobody.is_nobody?
 
     InfluxDB::Rails.current.tags = {
-      beta: !anonymous && User.current.in_beta?,
+      beta: !anonymous && User.possibly_nobody.in_beta?,
```

Comment:
Can the nobody user be in beta?

Reply:
No, it can't - but the current user can be nobody.